include/cantera/oneD/Flow1D.h

```diff
@@ -670,41 +674,166 @@ class Flow1D : public Domain1D
+     * using upwind differencing.
+     *
+     * The general formula for a variable, Var, is:
+     * @f[
+     *   \frac{\partial Var}{\partial z} \bigg|_{j} \approx \frac{Var(x, j_{\text{loc}}) -
+     *   Var(x, j_{\text{loc}} - 1)}{m_{\text{dz}}[j_{\text{loc}} - 1]}
+     * @f]
+     *
+     * Where the value of loc is determined by the sign of the axial velocity.
+     * If the axial velocity is positive, the value of loc is j. If the axial velocity
+     * is negative, the value of loc is j + 1. A positive velocity means that the flow
+     * is moving left-to-right.
+     *
+     * @param[in] x The local domain state vector.
+     * @param[in] j The index at which the derivative is computed.
+     */
     double dVdz(const double* x, size_t j) const {
-        size_t jloc = (u(x,j) > 0.0 ? j : j + 1);
-        return (V(x,jloc) - V(x,jloc-1))/m_dz[jloc-1];
+        size_t jloc = (u(x, j) > 0.0 ? j : j + 1);
+        return (V(x, jloc) - V(x, jloc-1))/m_dz[jloc-1];
     }
 
+    /**
+     * Calculates the spatial derivative of the species mass fraction @f$ Y_k @f$ with
+     * respect to z for species k at point j using upwind differencing.
+     *
+     * For details on the upwinding scheme, @see dVdz().
+     *
+     * @param[in] x The local domain state vector.
+     * @param[in] k The species index.
+     * @param[in] j The index at which the derivative is computed.
+     */
     double dYdz(const double* x, size_t k, size_t j) const {
-        size_t jloc = (u(x,j) > 0.0 ? j : j + 1);
-        return (Y(x,k,jloc) - Y(x,k,jloc-1))/m_dz[jloc-1];
+        size_t jloc = (u(x, j) > 0.0 ? j : j + 1);
+        return (Y(x, k, jloc) - Y(x, k, jloc-1))/m_dz[jloc-1];
     }
 
+    /**
+     * Calculates the spatial derivative of temperature T with respect to z at point
+     * j using upwind differencing.
+     *
+     * For details on the upwinding scheme, @see dVdz().
+     *
+     * @param[in] x The local domain state vector.
+     * @param[in] j The index at which the derivative is computed.
+     */
     double dTdz(const double* x, size_t j) const {
-        size_t jloc = (u(x,j) > 0.0 ? j : j + 1);
-        return (T(x,jloc) - T(x,jloc-1))/m_dz[jloc-1];
+        size_t jloc = (u(x, j) > 0.0 ? j : j + 1);
+        return (T(x, jloc) - T(x, jloc-1))/m_dz[jloc-1];
     }
     //! @}
 
+    /**
+     * Compute the shear term from the momentum equation using a central
+     * three-point differencing scheme. This term is discretized
+     *
+     * The term to be discretized is:
+     * @f[
+     *   \frac{d}{dz}\left(\mu \frac{dV}{dz}\right)
+     * @f]
+     *
+     * Let @f$ A = \mu \frac{dV}{dz} @f$ for simplicity. We'll call this the inner
+     * term or inner discretization. In this situation, the inner term is evaluated
+     * using a central difference formula, but evaluated at j+1/2 and j-1/2 (halfway
+     * between the grid points around point j).
+     *
```

Comment:
Again, I think it would be better to just link to what you've written in `discretization.md`. For the specific methods here, I think it's good to have the original (differential) form of the term that's being discretized, and that's really enough.

Reply:
I'd like to keep the discretization near the method itself. I view the documentation as downstream of the code primarily. So a referenced to the docs for the subtleties of the derivation, but the final discretized form being present in the docstring also.